go/libraries/doltcore/sqle/dolt_diff_summary_table_function.go

```diff
@@ -259,39 +302,40 @@ func (ds *DiffSummaryTableFunction) RowIter(ctx *sql.Context, row sql.Row) (sql.
 // evaluateArguments returns fromCommitValStr, toCommitValStr and tableName.
```

Comment:
Would be good to update this doc text, especially since the return types aren't very self-descriptive (`interface{}`, `string`).